### ISSUE TITLE
[TS] placeholder property missing on Input.Date and Input.Time in designer, and also not uses as aria-label in renderer

### ISSUE THREAD

**Issue by dclaux** [microsoft/AdaptiveCards#3410]
# Platform

TypeScript, all versions

# Details

The placeholder property is not exposed for Input.Date and Input.Time in the designer
The placeholder property should be used for the aria-label on those inputs.

**Comment by paulcam206** [comment]
Addressing this issue is not currently on our roadmap, so closing. Please feel free to continue the conversation in this issue for future consideration.
